frontend/micro-ui/web/micro-ui-internals/packages/modules/im/src/pages/employee/ComplaintDetails.js

```
@@ -762,104 +786,99 @@ export const ComplaintDetails = (props) => {
-              <MapView onClick={zoomView} />
-            </MediaRow>
-          )}
-        </StatusTable>
-      )}
-      {imagesToShowBelowComplaintDetails?.thumbs ? (
-        <div>
-        <CardLabel style={{marginTop:'18px', fontWeight:'bolder'}}>{t("CS_TICKET_ADDITIONAL_DETAILS")}</CardLabel>
-        <DisplayPhotos
-          srcs={[...(imagesToShowBelowComplaintDetails?.fullImage || []), ...(imagesToShowBelowComplaintDetails?.videos || [])]}
-          onClick={(source, index) => zoomImageWrapper(source, index)}
+            {1 === 1 ? null : (
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)